xds/src/main/java/io/grpc/xds/LoadReportClient.java

```diff
@@ -245,6 +239,10 @@ private void handleResponse(LoadStatsResponseData response) {
       if (closed) {
         return;
       }
+      if (!initialResponseReceived) {
+        logger.log(XdsLogLevel.DEBUG, "Initial LRS response received");
+        initialResponseReceived = true;
+      }
```

Comment:
Why not put the other one together?

Reply:
I don't understand your question. The other one is moved.

Reply:
I mean why not just move the whole 

```
if (!initialResponseReceived) {
        logger.log(XdsLogLevel.DEBUG, "Received LRS initial response:\n{0}", response);
        initialResponseReceived = true;
      } else {
        logger.log(XdsLogLevel.DEBUG, "Received LRS response:\n{0}", response);
      }
```
to where the proto is received instead of splitting them to two places.

Reply:
Ah, checking `initialResponseReceived` can only be in `syncContext`.